core/src/main/java/org/apache/brooklyn/core/mgmt/persist/DeserializingClassRenamesProvider.java

```diff
@@ -54,12 +55,20 @@
     }
     
     private static Map<String, String> loadDeserializingClassRenamesCache() {
-        InputStream resource = new ResourceUtils(DeserializingClassRenamesProvider.class).getResourceFromUrl(DESERIALIZING_CLASS_RENAMES_PROPERTIES_PATH);
-        if (resource != null) {
+        return ImmutableMap.<String, String>builder()
+                .putAll(loadDeserializingClassRenamesCacheFromUrl(DESERIALIZING_CLASS_RENAMES_PROPERTIES_PATH))
+                .putAll(loadDeserializingClassRenamesCacheFromUrl(KARAF_DESERIALIZING_CLASS_RENAMES_PROPERTIES_PATH))
```

Comment:
`ImmutableMap.Builder#putAll` for duplicate entries will fail.
It is good to be able to override `DESERIALIZING_CLASS_RENAMES_PROPERTIES_PATH` with the ones provided in the file `KARAF_DESERIALIZING_CLASS_RENAMES_PROPERTIES_PATH`.